README.md

```diff
@@ -171,630 +171,476 @@ curl https://raw.githubusercontent.com/grafana/helm-charts/main/charts/loki-stac
-[
-  {
-    "id": "230926ab-9483-4ba5-aa1a-f7850f63cxxx",
-    "name": "a7a979feb06504072a66238a105756b8",
-...
-```
-
-With proxy protocol enabled, the curl to the service fails.
+* In general, email-of-registrant is mandatory when using ACME: it should be a valid email address that will reach someone responsible for certificate management.
 
+* ACME stores certificates in Kubernetes secrets. The name of the secret can be set using the tlsSecret element:
 ```
-~ curl -i http://$LB_ENDPOINT/echo/         
-HTTP/1.1 400 Bad Request
-content-length: 11
-content-type: text/plain
-date: Mon, 28 Jun 2021 05:46:21 GMT
-server: envoy
-connection: close
 
-Bad Request%                                                                                                                  ~ 
-```
-
-That is because we need to tell Emissary ingress to treat the traffic coming in with a proxy protocol header.
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: quote-host
+spec:
+  hostname: quote.mandrakee.xyz
+  acmeProvider:
+    email: quote@gmail.com
+  tlsSecret:
+    name: tls-cert
+  requestPolicy:
+    insecure:
+       action: Redirect
+       additionalPort: 8080
 
-```
-kubectl apply -f - <<EOF
 ---
-apiVersion: x.getambassador.io/v3alpha1
-kind: AmbassadorListener
+apiVersion: getambassador.io/v2
+kind: Host
 metadata:
-  name: emissary-ingress-listener-8080
-  namespace: emissary
+  name: echo-host
 spec:
-  port: 8080
-  protocol: HTTPPROXY
-  securityModel: XFP
-  hostBinding:
-    namespace:
-      from: ALL
-EOF
+  hostname: echo.mandrakee.xyz
+  acmeProvider:
+    email: echo@gmail.com
+  tlsSecret:
+    name: tls2-cert
+  requestPolicy:
+    insecure:
+       action: Redirect
+       additionalPort: 8080
 
 ```
 
-Now I see the actual client IP at the pod.
+It takes ~30 seconds to get the signed certificate for the hosts. At this point, we have Ambassador installed, and hosts configured. BUT, we still do not have the networking (eg. DNS and Load balancer) configured to route the traffic to the cluster. 
 
-```
-~ curl -i http://$LB_ENDPOINT/echo/
-HTTP/1.1 200 OK
-content-type: text/plain
-date: Mon, 28 Jun 2021 05:48:48 GMT
-content-length: 359
-x-envoy-upstream-service-time: 5
-server: envoy
-
-Request served by echo-deployment-869b7bf9c7-r5mmd
-
-HTTP/1.1 GET /
-
-Host: 164.90.247.161
-Accept: */*
-X-Envoy-External-Address: 73.162.138.xxx
-X-Request-Id: 04e0e2a8-8b3b-474d-bc01-7a68d0bb01ed
-User-Agent: curl/7.64.1
-X-Forwarded-For: 73.162.138.xxx
-X-Forwarded-Proto: http
-X-Envoy-Expected-Rq-Timeout-Ms: 3000
-X-Envoy-Original-Path: /echo/
-Content-Length: 0
 
-```
+### Configure your domain mapping to point the domains to the cluster LB
 
-#### Getting traffic in using DNS name
-This is fairly straightforward. We will just add an A record for every host into the DNS. In my case, I added 2 hosts (echo and test) pointing to the same LB.
+Here  hosting 2 hosts (echo.mandrakee.xyz, quote.mandrakee.xyz) on the same cluster.
```

Comment:
You can add the A-record either using doctl, or from the UI.
https://docs.digitalocean.com/products/networking/dns/how-to/manage-records/

Reply:
done